### ISSUE TITLE
Missing comment object in post object

### ISSUE THREAD

**Issue by henryzt** [UCLCSSA/UCLCSSA-API#6]
Comment object is missing from the post object response, all comments of a specific post should always be returned with that post when requested. Unless comments are going to be requested separately?

**Comment by jieyouxu** [comment]
In terms of the "feed", are comments immediately required? Polling comments along is quite expensive to say the least.

**Comment by henryzt** [comment]
That's true, thus in current php backend only the latest 3 comments of each post in feed is returned. Need a discussion regarding whether to display the comment or not in the new feed layout.
